docs/scanners.rst

```diff
@@ -81,8 +84,12 @@ Physical scanners
 .. _Unkn0wnCat: https://github.com/Unkn0wnCat
 .. _muued: https://github.com/muued
 .. _philpagel: https://github.com/philpagel
+.. _eingemaischt: https://github.com/eingemaischt
 
 .. [1] Scanners with API Integration allow to push scanned documents directly to :ref:`Paperless API <api-file_uploads>`, sometimes referred to as Webhook or Document POST.
+.. [2] Canon Multi Function Printers show strange behaviour over SMB. They close and reopen the file after every page. So it is neccessary to not directly scan to the consume 
+folder but to a tmp folder. Adding ``/usr/bin/find [tmp-folder] -mmin +3 -type f -exec sh -c 'mv "{}" [consume-folder]$(uuidgen)-"$(basename "{}")"' \;`` to a crontab looks
+every minute for files that not have been written to for more than 3 minutes (scanner timeout) - and then moves them to the consume folder.
```

Comment:
Regarding this footnote, I think one of our newer configuration options `PAPERLESS_CONSUMER_POLLING_DELAY=180` might be the solution here. @stumpylog can you confirm that function would work as a replacement for the crontab entry?

Reply:
Some combination of the delays we have in now should be workable to prevent consumption too early.  

Maybe change this to be a note pointing to the configuration-polling and configuration-inotify links, saying these value should be tuned for your scanner?

Reply:
The polling_delay would only reduce the risk of a too early consumption - find only finds files which weren't altered for at least 3 minutes - and this is the important thing here.

Reply:
Oh, nevermind - I mixed it up with `PAPERLESS_CONSUMER_POLLING`. Thanks!

Reply:
PAPERLESS_CONSUMER_POLLING_DELAY sets the time between checks for the file to remain unmodified, as determined by the modified time and file size.  

Meanwhile, PAPERLESS_CONSUMER_POLLING_RETRY_COUNT sets the number of times the file will be checked for being unmodified.

The documentation for these was only recently added, though the functionality exists already.

Reply:
Yes => https://github.com/paperless-ngx/paperless-ngx/blob/dev/docs/configuration.rst

You are really doing a great job, thank you!